### ISSUE TITLE
Games that can be set past native resolution

### ISSUE THREAD

**Issue by justalex95** [Electry/VitaGrafixPatchlist#52]
I've found some games that can be set past native resolution, I havent fully completed all of these games but none of them have crashed at any of the resolutions detailed below. here they are...

- Street Fighter IV: 1480x720
- Trails of Cold steel: 960x720 (slowdown in school area and some cutscenes and special move effects but playable)
- Trails of Cold steel 2: 1280x544
- P4G: 960x720, MSAAx2
- World of Final Fantasy: 1280x544, effects stop working if set any higher
- Ninja Gaiden sigma 2+: 1280x544, vertical res cant be changed
- Soul Sacrifice Delta: 61440x34560 this isnt a joke, its not reporting the resolution correctly but you'll notice the difference
- Tokyo Xanadu: 1480x720, can go higher but has slowdown

These are the settings I found thats the best balance for me, as always, thanks Electry for vitagrafix :)

**Comment by dkrmdi** [comment]
Seems like Soul Sacrifice Delta Asian version does not work with that resolution. The game does not boot.

**Comment by Tellilum** [comment]
Dragon Quest Builders (US) Can be set to 1280x720p there’s a slowdown but its playable

**Comment by EnekoT2001** [comment]
Btw are you using PSVShell/lolicon to set the clocks speeds to max settings for testing? Bc maybe this way the ones that have some sort of slow down won't longer have slowdowns.

**Comment by Tellilum** [comment]
Yes. I’m using Lolicon set to Max. The slowdown may be due to multiple artifacts. After all, there’s quite abit going on in DQB

**Comment by justalex95** [comment]
Here, try 960x900, its slightly faster and gets rid of alot of the saw tooth on some blocks.

Also, try this out, I find it makes gameplay slightly smoother, not faster but smoother which makes the graphics smoother because the FPS is more fluid.

all you do is put 64504x64504 or 960x64504 after 960x900. a good game to test this on is killzone. it affects the speed of looking around and makes it look and play smoother. on killzone I prefer 960x64504.

You might not see a difference and thats cool, but if you can see a difference, awesome.

NOTE:This does not affect resolution in any way.

**Comment by justalex95** [comment]
Really enjoying this game, would never have picked it up otherwise what a great plugin!

**Comment by petecolon1985** [comment]
> I've found some games that can be set past native resolution, I havent fully completed all of these games but none of them have crashed at any of the resolutions detailed below. here they are...
> 
> * Street Fighter IV: 1440x720
> * Trails of Cold steel: 960x720 (slowdown in school area and some cutscenes and special move effects but playable)
> * Trails of Cold steel 2: 1280x544
> * P4G: 960x720, MSAAx2 or 960x911, MSAAx0
> * World of Final Fantasy: 1280x544, effects stop working if set any higher
> * Ninja Gaiden sigma 2+: 1280x544, vertical res cant be changed
> * Soul Sacrifice Delta (not ASIA ver): 61440x34560 this isnt a joke, its not reporting the resolution correctly but you'll notice the difference
> * Tokyo Xanadu: 1440x720, MSAAx4
> * Dragon Quest Builders: 1025x911 its the highest resolution I found while being pretty damn playable, with a good improvement in graphics fidelity and saw tooth reduction.
> 
> These are the settings I found thats the best balance for me, as always, thanks Electry for vitagrafix :)

Street Fighter IV for the ps vita? I was not aware of this game being on the vita, im confused

**Comment by justalex95** [comment]
haha i meant street fighter x tekken

**Comment by sumimasen03** [comment]
Im using us ver. If soul sd. Everytime i set with that reso. The game won't boot

**Comment by dkrmdi** [comment]
> Im using us ver. If soul sd. Everytime i set with that reso. The game won't boot

So I was not the only one. I don't know how some people say that reso works.

**Comment by justalex95** [comment]
> > Im using us ver. If soul sd. Everytime i set with that reso. The game won't boot
> 
> So I was not the only one. I don't know how some people say that reso works.

havent tested it with the new vitagrafix version as I completed it some time ago but...

https://imgur.com/4YNgJ6Q


it worked

**Comment by dkrmdi** [comment]
> > > Im using us ver. If soul sd. Everytime i set with that reso. The game won't boot
> > 
> > 
> > So I was not the only one. I don't know how some people say that reso works.
> 
> havent tested it with the new vitagrafix version as I completed it some time ago but...
> 
> https://imgur.com/4YNgJ6Q
> 
> it worked
> 
> EDIT: no longer works with new vitagrafix versions, if you want it to work use 4.0 alpha

Can you send me your patchlist? Maybe the patchlist I have is different from yours.

**Comment by justalex95** [comment]
the new version of vitagrafix works differently and has alot more features, the code has probably been optimised and may have broken the way it worked before.

[vitagrafix Alpha](http://s000.tinyupload.com/index.php?file_id=80261437672922311124)

here's the version i'm talking about, if you wnt to keep the patches for uncharted or mortal combat, paste them into your patchlist with this version and see if it works. 

I dont have the time to test it out right now but I dont see why that wouldnt work.

hope that helps

EDIT: SSD is one of the games that I didnt need to add to the patchlist so we'll be using the same code for it, so it should work

**Comment by dkrmdi** [comment]
> the new version of vitagrafix works differently and has alot more features, the code has probably been optimised and may have broken the way it worked before.
> 
> [vitagrafix Alpha](http://s000.tinyupload.com/index.php?file_id=80261437672922311124)
> 
> here's the version i'm talking about, if you wnt to keep the patches for uncharted or mortal combat, paste them into your patchlist with this version and see if it works.
> 
> I dont have the time to test it out right now but I dont see why that wouldnt work.
> 
> hope that helps
> 
> EDIT: SSD is one of the games that I didnt need to add to the patchlist so we'll be using the same code for it, so it should work

I think only the english version od SSD works. Asian version does not work.
